src/components/entry-boxes/entry-boxes.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Component from './entry-boxes-component';
-
+import { getCountedActiveLayers } from './entry-boxes-selectors';
```

Comment:
@aabdaab , @kfilip94 just opening this conversation.
I prefer the pattern we where doing lately of exporting `default` from selectors files an structured selector (`createStructuredSelector({ ...selectors map in here... }))`) an doing `import mapStateToProps from './selectors'` than importing the named selectors and creating the `mapStateToProps` on the entry file. I think the component container file it is cleaner and having the structured selector at the end of the selectors file also looks quite terse.
What do you think?

Reply:
I also prefer this way. If you create a `selectors` file anyway you might just as well export the `createStructuredSelector` there; this way it's a lot easier to add more props to the component without lots of changes

Reply:
I agree with you, it can be a cleaner approach. I'm going to change it now :+1: